site/content/docs/install_aio.md

```diff
@@ -0,0 +1,451 @@
+you use a **tmpfs** memory mount point for this.  Docker will not allow the container to mount tmpfs without having
+CAP\_SYS\_ADMIN capability (``-privileged``).  To work around this limitation, the container will not create the tmpfs.  Instead
+the container will use ``/var/mysqltmp`` which can be a volume to the host system tmpfs mount point.   The host system 
+should create a tmpfs and then map that as a volume in docker using ``-v /var/openbmp/mysqltmp:/var/mysqltmp``
+
+#### On host create mysql shared dir
+    mkdir -p /var/openbmp/mysql
+    chmod 777 /var/openbmp/mysql 
+
+> The mode of 777 can be changed to chown <user> but you'll have to get that ID 
+> by looking at the file owner after starting the container. 
+    
+
+#### On host create tmpfs (as root)
+
+    mkdir -p /var/openbmp/mysqltmp
+    echo "tmpfs /var/openbmp/mysqltmp tmpfs defaults,gid=nogroup,uid=nobody,size=2400M,mode=0777 0 0" >> /etc/fstab
+    mount /var/openbmp/mysqltmp
+
+### 4) Run docker container
+
+> #### Memory for MySQL
+> Mysql requires a lot of memory in order to run well.   Currently there is not a  consistent way to check on the container memory limit. The ```-e MEM=size_in_GB`` should be specified in gigabytes (e.g. 16 for 16GB of RAM).   If you fail to supply this variable, the default will use **/proc/meminfo** .  In other words, the default is to assume no memory limit. 
+
+#### Environment Variables
+Below table lists the environment variables that can be used with ``docker -e <name=value>``
+
+NAME | Value | Details
+:---- | ----- |:-------
+**API\_FQDN** | hostname | **required** Fully qualified hostname for the docker host of this container, will be used for API and Kafka. It is also the collector Admin Id
+MEM | RAM in GB | The size of RAM allowed for container in gigabytes. (e.g. ```-e MEM=15```)
+OPENBMP_BUFFER | Size in MB | Defines the openbmpd buffer per router for BMP messages. Default is 16 MB.  
+REINIT_DB | 1 | If set to 1 the DB will be reinitialized, which is needed to load the new schema sometimes.  This will wipe out the old data and start from scratch.  When this is not set, the old DB is reused.   (e.g. ```-e REINIT_DB=1```)
+MYSQL\_ROOT\_PASSWORD | password | MySQL root user password.  The default is **OpenBMP**.  The root password can be changed using [standard MySQL instructions](https://dev.mysql.com/doc/refman/5.6/en/resetting-permissions.html).  If you do change the password, you will need to run the container with this env set.
+MYSQL\_OPENBMP\_PASSWORD | password | MySQL openbmp user password.  The default is **openbmp**.  You can change the default openbmp user password using [standard mysql instructions](https://dev.mysql.com/doc/refman/5.6/en/set-password.html).  If you change the openbmp user password you MUST use this env.  
+
+#### Run normally
+> ##### IMPORTANT
+> Make sure to define **API_FQDN** as a hostname (or fqdn) and not by IP.  The hostname should
```

Comment:
I changed the format a little bit. Moved the "Run Normally" below the "IMPORTANT" and made that whole paragraph regular color. Let's see if that will make a difference.